pkg/nodeconfig/nodeconfig.go

```diff
@@ -190,7 +190,7 @@ func (nc *nodeConfig) Configure() error {
 			nodeConfigCache.credentials); err != nil {
 			return fmt.Errorf("configuring WICD failed: %w", err)
 		}
-		// Set the desired version annotation, communicating to WICD which Windows services configmap to use
+		// Set the desired version annotation
```

Comment:
Why is this being changed?

Reply:
I misunderstood its context (config flow vs deconfig flow), will revert